test/integration/proposal-checks/PostProposalCheck.sol

```diff
@@ -1,25 +1,31 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity 0.8.13;
 
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
 import {Test} from "@forge-std/Test.sol";
 
-import {Addresses} from "../../proposals/Addresses.sol";
-import {TestProposals} from "../../proposals/TestProposals.sol";
+import {Addresses} from "@test/proposals/Addresses.sol";
+import {TestProposals} from "@test/proposals/TestProposals.sol";
 
 contract PostProposalCheck is Test {
     Addresses addresses;
+    address public proposalZero;
```

Comment:
why add a reference to proposalZero here ? after the protocol is live, proposal 0 (protocol deployment) won't be in the array of executed proposals anymore, it will be replaced by entries in protocol-configuration/addresses.json

Reply:
seems that you're using it as a "protocol deployer that holds 100 rebasing CREDIT" reference in integration test `IntegrationTestBorrowSDAICollateral.sol`, I think it would be easy to remove this reference & add a rebasing user before these tests that need a rebasing user